### PR TITLE
Fix clang-format in various codes checked in recently.

### PR DESCRIPTION
**Pull request by whchung**
These files are causing other outstanding PRs from completing CI runs.

### CODE REVIEW

**Comment by krzysz00** [comment]
Isn't this fix for this to merge/rebase miopen-dialect so that these lines don't show up in the diff? They're presumably formatted correctly overall, otherwise they couldn't've been checked in themselves.

Unless this is a separate issue.

**Comment by whchung** [comment]
I was trying to remove `skip-ci` label on #514, and think these changes deserves to be in a separate PR, to keep #514 succinct. As you can see in the files changed here, they are indeed not formatted correctly. I have no idea why they went in undetected.

File: mlir/lib/CAPI/Dialect/MIGraphX.cpp [hunk 1]
```diff
@@ -33,7 +33,8 @@ void mlirGetKernelInfo(MlirModule module, int *size, void *data) {
   int argIdx = 0;
   llvm::StringRef kernelName;
 
-  assert (size != nullptr || data != nullptr); // either of ptrs should be provided.
+  assert(size != nullptr ||
```
Comment (krzysz00) [inline mlir/lib/CAPI/Dialect/MIGraphX.cpp]: Can we go ahead and make this a noisy assert? `assert((size != nullptr || data != nullptr) && "Either size or data pointer should be provided)`

**Review by krzysz00 — COMMENTED** [review]
Yeah, very weird that these didn't get caught

One nitpick here, then I'm willing to toss these in CI-less

File: mlir/lib/CAPI/Dialect/MIGraphX.cpp (hunk 1 above)
Comment (whchung) [reply mlir/lib/CAPI/Dialect/MIGraphX.cpp]: I'm merely fixing `clang-format` here. To improve the quality of the assert I think it's up to the owner of the module, @jungpark-mlir ?

Comment (jungpark-mlir) [reply mlir/lib/CAPI/Dialect/MIGraphX.cpp]: Yeah, that change (embedding comment) looks better

**Review by jungpark-mlir — APPROVED** [review]
Thanks for fixing this.

**Review by krzysz00 — APPROVED** [review]

**Pull request merged**